setup.cfg

```diff
@@ -67,7 +69,8 @@ prometheus =
 	prometheus-client<0.8;python_version>"3.5"
 flask =
 	flask~=1.1;python_version~="3.5.0"
-	flask<3;python_version>"3.5"
+	flask<3;python_version>"3.5" and python_version<"3.9"
+	flask>=3,<4;python_version>="3.9"
```

Comment:
This is gratuitously forcing users on recent python versions to use Flask 3. The Store is one such case for example :) running on Jammy (py310 ) on Flask 2.3.2. 

```sh
$ env/bin/pip install -e .[flask] flask==2.3.2
...
ERROR: Cannot install flask==2.3.2 and talisker[flask]==0.21.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested flask==2.3.2
    talisker[flask] 0.21.5 depends on flask<4 and >=3
...
```

Reply:
Can we maybe just `flask<=3` for any python version?

Reply:
Sure, we can be more relaxed in the requirements here and let downstream projects choose the version they want. I'll adjust it here for flask and the other requirements.